maddymz/irondb
# Us679 - User Panel w/ functionality

User Panel Functionality

To Test:
checkout US679
./iron.sh -p (re-initialize the database)
start server
navigate to localhost:3001/register
make up some credentials

You should automatically be redirected to the user-panel, which will dynamically insert mock data from the database. It should look like:

![image](https://user-images.githubusercontent.com/24575856/52893263-5c2ad880-314f-11e9-9e2d-343f13f18fd7.png)

Click the 'Reported Entries' and 'Pending Approval' links to make sure you are redirected to the correct page.
